docs/5.0/pages/installation.md

```diff
@@ -14,57 +14,24 @@ The following examples install the 64-bit version of Teleport binaries, but
 Release](https://gravitational.com/teleport/download/) page for the most
 up-to-date information.
 
-=== "Tarball"
+=== "DEB repo (Debian/Ubuntu/Linux Mint)"
 
     ```bash
-    $ curl https://get.gravitational.com/teleport-v{{ teleport.version }}-linux-amd64-bin.tar.gz.sha256
-    # <checksum> <filename>
-    $ curl -O https://get.gravitational.com/teleport-v{{ teleport.version }}-linux-amd64-bin.tar.gz
-    $ shasum -a 256 teleport-v{{ teleport.version }}-linux-amd64-bin.tar.gz
-    # Verify that the checksums match
-    $ tar -xzf teleport-v{{ teleport.version }}-linux-amd64-bin.tar.gz
-    $ cd teleport
-    $ ./install
-    $ which teleport
-    /usr/local/bin/teleport
+    # Install our public key. Fingerprint: 0c5e 8ba5 658e 320d 1b03 1179 c87e d53a 6282 c411
+    $ curl https://deb.releases.teleport.dev/teleport-pubkey.asc | sudo apt-key add -
+    # Add repo to APT
+    $ add-apt-repository 'deb https://deb.releases.teleport.dev/ stable main'
+    # Update APT Cache
+    $ apt update
```

Comment:
@webvictim  not sure if `apt` is installed by default on Debian and Mint, let's do `apt-get`